doc/man3/EVP_des_cbc.pod

```diff
@@ -54,6 +54,8 @@ EVP_des_ofb()
 DES in CBC, ECB, CFB with 64-bit shift, CFB with 1-bit shift, CFB with 8-bit
 shift and OFB modes.
 
+All these algorithms are available only in case when legacy provider is loaded.
```

Comment:
_All these algorithms are available only when the legacy provider is loaded._

I.e. delete _in case_ and add _the_.

Reply:
This is not true. Someone else might make a provider that includes these algorithms, it doesn't have to be our legacy provider.

Reply:
This is a case we don't have to document.  This is our documentation about our providers and core.